libversion.gradle

```diff
@@ -1 +1 @@
-ext.libversion = "9.2.0"
+ext.libversion = "9.3.0"
```

Comment:
Wrong version number?

Reply:
The lib version was already in 9.2.0 I just bump the version. I realize it is not supposed to expose the next version but the current one. I am not sure why the version has been bumped outside from a release PR.

Reply:
>  I am not sure why the version has been bumped outside from a release PR.

When PRs are merged, a -SNAPSHOT package is published. I bumped so that we have newer changes in a `next-version-SNAPSHOT` rather than in a `current-version-SNAPSHOT`.